python/tvm/script/parse/var_table.py

```diff
@@ -62,3 +62,9 @@ def add(self, var: str, value: Any):
 
     def get(self) -> Dict[str, Any]:
         return {key: values[-1] for key, values in self.name2value.items() if values}
+
+    def exist(self, value: Any):
+        for v in self.name2value.values():
+            if v is value:
```

Comment:
i'm not 100% sure whether or not we should use `is` operator here but I don't have better idea for now. shall we deliberate a little bit?